pkg/migration/controllers/migration.go

```diff
@@ -884,6 +989,10 @@ func (m *MigrationController) applyResources(
 		if !ok {
 			return fmt.Errorf("unable to cast object to unstructured: %v", o)
 		}
+
+		unstructured.SetAnnotations(map[string]string{StorkMigrationAnnotation: "true"})
```

Comment:
This will wipe out all the other annotations in the objects. You have the `metadata` for the object above. Get the annotations for the objects and add this to those.

As a separate change, we should also add an annotation with the timestamp and name of the migration.

Reply:
good catch , will update